library/phpstan-baseline.neon

```diff
@@ -3725,6 +3730,11 @@ parameters:
 			count: 1
 			path: Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php
 
+		-
+			message: "#^Parameter \\#1 \\$startTime of method Ivoz\\\\Provider\\\\Domain\\\\Model\\\\BillableCall\\\\BillableCallDtoAbstract\\:\\:setStartTime\\(\\) expects DateTimeInterface\\|string, DateTimeInterface\\|string\\|null given\\.$#"
+			count: 1
+			path: Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php
+
```

Comment:
I'm pretty sure you can fix this errors without ignoring them

Reply:
Not that easy: I should modify generated code (like BillableCallDtoAbstract.php) or ignore it with a comment in code not directly related to this PR:
https://github.com/irontec/ivozprovider/blob/2bd8decc33d782ce69edf94c6ffb06365c346bbf/library/Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdr.php#L46-L49